### DISCUSSION TITLE
Site localization into Russian

### DISCUSSION THREAD

**Discussion by rakleed** [community/community#8150] [Other Feature Feedback, Questions, & Ideas]
Hello, could you add Russian localization to your site? It would be very helpful for Russian-speaking users.

**Comment by vaultdude** [comment]
@ettaboyle Hello there!
I often see you almost everywhere giving feedback about the wishes here, I just wanted to remind you that this introduction to the interface would make the site for young professionals from my country more friendly. Right now I'm taking a course on web development for beginners, there are about 250 people in our group and it's hard for many to master your site for practice (unfortunately, our methodologies use only github).
Thanks.
